javascript/src/views/keypad-universe.html

```diff
@@ -0,0 +1,8 @@
+<ul class="nav nav-tabs">
```

Comment:
Tuckie linked to this too http://getbootstrap.com/components/#nav-tabs . Is there no tool as part of the build scripts you've got that can append and set which tab is active for each file. Perhaps it just needs a little bit of Python or something, but it seems very error prone having to do all the copy/pasting and editing each time you add a new tab.